CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ endif()
 option(EXTLIB_FROM_SUBMODULES "Use Git submodules for header-only dependencies" OFF)
 option(${PROJECT_NAME}_CXX_WARNINGS "Compile C++ with warnings as errors" ON)
 option(REPORTS_ENABLE_MPI "Enable MPI-based execution" ON)
+option(SONATA_REPORT_TEST "Enable tests" ON)
```

Comment:
can we have consistent options naming?
e.g. `SONATA_REPORT_ENABLE_MPI`, `SONATA_REPORT_ENABLE_TEST`, `SONATA_REPORT_ENABLE_SUBMODULES` etc. or something similar.